### ISSUE TITLE
check_ltlspec_klive Invalid command

### ISSUE THREAD

**Issue by sloth0830** [relentless-warrior/5GReasoner#2]
README for nuXmv/Isolation states to run "check_ltlspec_klive" which is not a valid command. Running Check_ltlspec causes system to hang. 

Running on WSL Ubuntu 20 w/ last Python2 package.

**Comment by xxixi8** [comment]
@sloth0830  @relentless-warrior 
I have the problem as you. Have you solved it?

**Comment by sloth0830** [comment]
@xixihahahha I was able to resolve, the NuXmv changed the command I was able to find it in older version of the NuXmv manual. (This repo is ~5 years old) 

Now use Check_ltlspec_ic3 -K 1

![image](https://github.com/relentless-warrior/5GReasoner/assets/57303675/47dd9f01-520d-409d-a406-0f610ce6ec0e)

**Comment by xxixi8** [comment]
@sloth0830 
Thanks. I have solved it. I would like to ask which version of the NuXmv  manual you refer to？I can only find the latest verson of the manual on the official website.

**Comment by cici0410** [comment]
> 谢谢。我已经解决了。我想问一下你参考的是哪个版本的NuXmv手册？我只能在官方网站上找到手册的最新版本。

Hello, I would like to ask you, how long does it take to run the build_boolean_model instructions in the NAS layer in Nuxmv, I have been running for a long time and have not been able to wait for the result

**Comment by cici0410** [comment]
> @sloth0830 Thanks. I have solved it. I would like to ask which version of the NuXmv manual you refer to？I can only find the latest verson of the manual on the official website.

Hello, did you also solve the problem by running the 'Check_ltlspec_ic3 -K 1' command? I ran it on Windows, but the system displayed that the command was invalid.

**Comment by xxixi8** [comment]
> > @sloth0830 Thanks. I have solved it. I would like to ask which version of the NuXmv manual you refer to？I can only find the latest verson of the manual on the official website.
> 
> Hello, did you also solve the problem by running the 'Check_ltlspec_ic3 -K 1' command? I ran it on Windows, but the system displayed that the command was invalid.

@cici0410 yes, the This command should be in lower case，'check_ltlspec_ic3 -K 1'
